src/directives/mlhrTable.js

```diff
@@ -188,7 +187,34 @@ angular.module('datatorrent.mlhrTable.directives.mlhrTable', [
     }
 
     var scrollDeferred;
-    var debouncedScrollHandler = debounce(function() {
+    var scrollTopSaved = -1;
+
+    // determine requestAnimationFrame compabitility
+    var raf = window.requestAnimationFrame
+            || window.mozRequestAnimationFrame
+            || window.webkitRequestAnimationFrame
+            || window.msRequestAnimationFrame
+            || function(f) { return setTimeout(f, 100) };
```

Comment:
@peyao Good catch, I'll update it to use the debounce option value.  It's a fallback function anyway, but will use it.

I didn't mention in the PR, but I can to update the documentation and unit tests for this change, but I want to get the acceptance of feature and code change before I do that.